src/actions/index.js

```diff
@@ -48,6 +62,10 @@ const apiAction = (method, dataKey, options) => _dispatch => {
         })
 
         try {
+            if (isNetworkAction(method) && !apiOptions.isClient) {
+                return await _dispatch()
```

Comment:
It might be OK, but I'm a bit concerned of what side effects this may introduce..

Reply:
well this is interesting! i'm debugging a weird error we're seeing in Sentry, and i've narrowed it down to undefined actions being dispatched: https://github.com/Patreon/patreon_react_features/issues/20339

do we think this could be related?